### PR TITLE
Add support for postDeployTask

### PR DESCRIPTION
**Pull request by EricJizbaMSFT**
Related to https://github.com/microsoft/vscode-azurefunctions/issues/1528

### CODE REVIEW

File: appservice/src/index.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ export * from './deleteSite';
 export * from './deploy/deploy';
 export * from './deploy/getDeployFsPath';
 export * from './deploy/IDeployContext';
-export * from './deploy/runPreDeployTask';
+export { handleFailedPreDeployTask, IPreDeployTaskResult, runPreDeployTask, tryRunPreDeployTask } from './deploy/runDeployTask';
```
Comment (nturinski) [inline appservice/src/index.ts]: Why are you exporting `handleFailedPreDeployTask`?  I don't see it used in https://github.com/microsoft/vscode-azurefunctions/pull/1996/files

Comment (EricJizbaMSFT) [reply appservice/src/index.ts]: It's used here: https://github.com/microsoft/vscode-azurefunctions/blob/master/src/commands/deploy/runPreDeployTask.ts#L35

All the functions listed here already existed and were exported previously. The only thing _not_ exported out of `runDeployTask.ts` is `startPostDeployTask` which is the function I just added

Comment (nturinski) [reply appservice/src/index.ts]: Oh doh, you were * exporting.  Sorry 😬

Comment (EricJizbaMSFT) [reply appservice/src/index.ts]: dontworry

**Pull request merged**